lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -1127,46 +1146,13 @@ public void setApplyingOpacityToLayersEnabled(boolean isApplyingOpacityToLayersE
   }
 
   /**
-   * Disable the extraScale mode in {@link #draw(Canvas)} function when scaleType is FitXY. It doesn't affect the rendering with other scaleTypes.
-   *
-   * <p>When there are 2 animation layout side by side, the default extra scale mode might leave 1 pixel not drawn between 2 animation, and
-   * disabling the extraScale mode can fix this problem</p>
-   *
-   * <b>Attention:</b> Disable the extra scale mode can downgrade the performance and may lead to larger memory footprint. Please only disable this
-   * mode when using animation with a reasonable dimension (smaller than screen size).
+   * This API no longer has any effect.
    */
+  @Deprecated
```

Comment:
How do you normally handle this? When it comes a major version you will remove it from the codebase?

Reply:
@sanogueralorenzo I don't think I ever have before but probably as you said.